package.json

```diff
@@ -85,7 +85,10 @@
     "source-map-generator": "0.8.0"
   },
   "tsd": {
-    "directory": "test/types"
+    "directory": "test/types",
+    "compilerOptions": {
+      "types": ["jest"]
```

Comment:
There's no way (that I can find) to tell `tsd` to use anything other than `tsconfig.json`, so we have to override the `types` field here. 

It might have been cleaner to keep `tsconfig.json` as the "build" config, and use `tsconfig-lint.json` for vscode's typescript validation. But vscode doesn't support using a different file name either.